vector/src/androidTest/java/im/vector/app/ui/robot/OnboardingRobot.kt

```diff
@@ -34,31 +34,46 @@ import im.vector.app.waitForView
 
 class OnboardingRobot {
 
+    private val defaultVectorFeatures = DefaultVectorFeatures()
+
     fun crawl() {
         waitUntilViewVisible(withId(R.id.loginSplashSubmit))
-        crawlGetStarted()
+        crawlCreateAccount()
         crawlAlreadyHaveAccount()
     }
 
-    private fun crawlGetStarted() {
-        clickOn(R.id.loginSplashSubmit)
-        assertDisplayed(R.id.useCaseHeaderTitle, R.string.ftue_auth_use_case_title)
-        clickOn(R.id.useCaseOptionOne)
-        OnboardingServersRobot().crawlSignUp()
-        pressBack()
-        pressBack()
+    private fun crawlCreateAccount() {
+        if (defaultVectorFeatures.isOnboardingCombinedRegisterEnabled()) {
+            // TODO
```

Comment:
is there an issue in GH to mention here? If not, it worth at least to mention when it's planned to be implemented or why we can't implement it now

Reply:
good point! will create an issue and link in the code :+1:

Reply:
https://github.com/vector-im/element-android/issues/6652